.github/workflows/ci-python.yml

```diff
@@ -1,38 +1,36 @@
+---
```

Comment:
What's this? 👀

Reply:
These dashes mark the beginning of a new YAML document. Can be used to have multiple documents in one file. In this case they were added automatically by my formatter.

Reply:
While this might be syntactically cleaner, I see little benefit for this use case and it seems to be very uncommon (less than 4% of GitHub workflow files seem to use this, see [this](https://github.com/search?q=+language%3Ayaml+path%3A%2F%5E%5C.github%5C%2Fworkflows%5C%2F%2F+%2F%5E---%2F&type=code) vs [this](https://github.com/search?q=+language%3Ayaml+path%3A%2F%5E%5C.github%5C%2Fworkflows%5C%2F%2F&type=code)).
Any thougts, @mbrinkhoff?

Reply:
removed in 0b6c64b71bb6d7139b43535c0c8c4f06c961c8f3